app/scripts/modules/core/src/widgets/AccountRegionClusterSelector.tsx

```diff
@@ -0,0 +1,23 @@
+import { Application, IAccount } from 'core';
+
+export interface IAccountRegionClusterSelectorProps {
+  application: Application;
+  component: Object;
+  accounts: IAccount[] | string[];
+  clusterField?: string;
+  singleRegion?: string;
+  showAllRegions?: boolean;
+  onAccountUpdate?: (account: string) => void;
+  disableRegionSelect?: boolean;
+}
+
+export const accountRegionClusterSelectorBindings: Record<keyof IAccountRegionClusterSelectorProps, string> = {
+  application: '<',
+  component: '<',
+  accounts: '<',
+  clusterField: '<',
+  singleRegion: '<',
+  showAllRegions: '<',
+  onAccountUpdate: '<',
```

Comment:
This won't work since the actual angular binding is `&`. You'll have to create a wrapper component that translates the binding. See https://github.com/spinnaker/deck/blob/eeda562a8c2c848ecfb81c98fb0a643a578d3f7e/app/scripts/modules/core/src/forms/numberList/numberList.component.ts#L144-L164

Reply:
yes, i've corrected the PR